pkg/apis/machine/v1alpha1/machineclass_types.go

```diff
@@ -73,20 +74,17 @@ type MachineClassList struct {
 // ExpectedNodeAttributes contains subfields to track all scale from zero resources
 type ExpectedNodeAttributes struct {
 	// CPU cores of the expected node
-	// +optional
-	CPU *int64 `json:"cpu,omitempty"`
+	CPU resource.Quantity `json:"cpu,omitempty"`
```

Comment:
we can remove `omitempty` from required fields

Reply:
Yes